Geometry/HGCalCommonData/interface/HGCalDDDConstants.h

```diff
@@ -24,7 +24,7 @@ class HGCalDDDConstants {
 
 public:
 
-  HGCalDDDConstants(const HGCalParameters* hp, const std::string name);
+  HGCalDDDConstants(const HGCalParameters* hp, std::string name);
```

Comment:
@davidlange6 - this will be fixed in the next PR:
```
$ git diff Geometry/HGCalCommonData/interface/HGCalDDDConstants.h
diff --git a/Geometry/HGCalCommonData/interface/HGCalDDDConstants.h b/Geometry/HGCalCommonData/interface/HGCalDDDConstants.h
index 87c03ab..22f8f2d 100644
--- a/Geometry/HGCalCommonData/interface/HGCalDDDConstants.h
+++ b/Geometry/HGCalCommonData/interface/HGCalDDDConstants.h
@@ -24,7 +24,7 @@ class HGCalDDDConstants {
 
 public:
 
-  HGCalDDDConstants(const HGCalParameters* hp, const std::string name);
+  HGCalDDDConstants(const HGCalParameters* hp, const std::string& name);
   ~HGCalDDDConstants();
 
   std::pair<int,int>  assignCell(float x, float y, int lay, int subSec,
```